### PR TITLE
Feat/1208 external db facets as 'not missing' query

### PR DESCRIPTION
**Pull request by cy**
depends on https://github.com/NCI-GDC/gdcapi/pull/788

### CODE REVIEW

**Comment by cy** [comment]
this is ready for review. Adds `NotMissingFacet` component under Mutation facet panel that sets a `not missing` query. Does not actually use aggregations for counts because ran into 100 buckets size limit, so using filtered hits total.

File: modules/node_modules/@ncigdc/utils/__tests__/string.test.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import { capitalize } from '../string';
+
+describe('capitalize', () => {
+  it('should captalize a string', () => {
+    expect(capitalize('danish marzipan biscuit')).toEqual('Danish Marzipan Biscuit');
+  });
+
+  it('should capitalize mirna as miRNA', () => {
+    expect(capitalize('mirna')).toEqual('miRNA');
+    expect(capitalize('danish marzipan mirna biscuit')).toEqual('Danish Marzipan miRNA Biscuit');
```
Comment (alex-wilmer) [inline modules/node_modules/@ncigdc/utils/__tests__/string.test.js]: 🍪

Comment (cy) [inline modules/node_modules/@ncigdc/utils/__tests__/string.test.js]: mmm miRNA biscuits

**Comment by cy** [comment]
@alex-wilmer you mean these https://github.com/NCI-GDC/portal-ui/pull/1687/files#diff-f3e533b9b48faa3f3cb23696da58d47dR162 filters? Don't expect those to change, so I think it's ok having them directly in the string.

**Pull request merged**